project-management/src/main/java/life/qbic/projectmanagement/application/ProjectOverview.java

```diff
@@ -23,6 +22,7 @@
  */
 //Views are aggregates which cannot be changed
 @Immutable
+@org.springframework.data.annotation.Immutable
```

Comment:
Now the project overview has both immutable annotations? I'd expected the "wrong" immutable to be replaced?

Reply:
It might be good to have both in here so that both frameworks know of the immutability